src/components/progress-circle/progress-circle.ts

```diff
@@ -250,8 +250,8 @@ function clamp(v: number) {
  * Returns the current timestamp either based on the performance global or a date object.
  */
 function now() {
-  if (window.performance && window.performance.now) {
-    return window.performance.now();
+  if (typeof performance !== 'undefined' && performance.now) {
```

Comment:
Ah, in #837 the user mentioned that there's a runtime exception when rendering server-side, because of the references to `window`.